plugins/forum/README.md

```diff
@@ -0,0 +1,18 @@
+FoodsoftForum
+=================
+
+This plugin adds a forum to foodsoft. A new 'Forum' menu entry is added below the 'Foodcoops' menu in the navigation bar.
+
+This plugin is enabled by default in foodsoft, so you don't need to do anything
+to install it. If you still want to, for example when it has been disabled,
+add the following to foodsoft's Gemfile:
+
+```Gemfile
+gem 'foodsoft_forum', path: 'lib/foodsoft_forum'
+```
```

Comment:
and then also include the forum gems here. It's not enough that they are mentioned in the gemspec, because we need the github version.